src/syft/lib/torch/allowlist.py

```diff
@@ -617,3 +617,10 @@
 # allowlist.add("torch.nn.Linear.parameters")
 allowlist["torch.nn.parameter.Parameter"] = "torch.nn.parameter.Parameter"
 allowlist["torch.nn.parameter.Parameter.__add__"] = "torch.nn.parameter.Parameter"
+
+
+# MNIST
+allowlist["torch.manual_seed"] = "torch.Generator"
+allowlist["torch.Generator"] = "torch.Generator"
+allowlist["torch.Generator.get_state"] = "torch.Tensor"  # Probably dont want this?
```

Comment:
No reason to not include it :)

Reply:
Is there any thing insecure about getting or setting the state of a random number generator?